WordPress/Classes/ViewRelated/Post/PostPreviewViewController.m

```diff
@@ -13,14 +13,16 @@
 
 
 
-@interface PostPreviewViewController () <PostPreviewGeneratorDelegate>
+@interface PostPreviewViewController () <PostPreviewGeneratorDelegate, WPNoResultsViewDelegate>
 
 @property (nonatomic, strong) UIWebView *webView;
 @property (nonatomic, strong) NSMutableData *receivedData;
 @property (nonatomic, strong) AbstractPost *apost;
 @property (nonatomic, strong) UIBarButtonItem *shareBarButtonItem;
 @property (nonatomic, strong) UIBarButtonItem *doneBarButtonItem;
 @property (nonatomic, strong) PostPreviewGenerator *generator;
+@property (nonatomic, strong) WPNoResultsView *noResultsView;
```

Comment:
Thank you! It would be nice to use the new one. If for some reason it doesn't work for this case, next week I'll be replacing WPNoResultsView globally, so I can fix it then. But it would be nice if I didn't have to. 😀